### PR TITLE
[one-cmds] Add ampq option to one-quantize

### PR DESCRIPTION
**Pull request by stamalakhov**
This commit implements ampq option processing in one-quantize.

Its correctness is tests at #11179.

Draft: #11179
Related: #11146

ONE-DCO-1.0-Signed-off-by: s.malakhov <s.malakhov@partner.samsung.com>

### CODE REVIEW

File: compiler/one-cmds/one-quantize [hunk 1]
```diff
@@ -226,6 +226,29 @@ def _get_parser():
...
+        '--ampq_algorithm', type=str, help='type of algorithm (bisection)')
+
+    ampq_quant_group.add_argument(
+        '--bisection_type', type=str, help="one of 'auto', 'i16_front', 'i16_back'")
+
+    # ampq_bisection_visq
+    ampq_quant_group.add_argument(
+        '--ampq_bisection_visq',
+        type=str,
+        help='.visq.json file with quantization errors')
```
Comment (seanshpark) [inline compiler/one-cmds/one-quantize:250]:
```suggestion
        help='.visq.json file path with quantization errors')
```

**Comment by jinevening** [comment]
@nnfw-bot test nncc-debug

**Review by seanshpark — APPROVED** [review]
LGTM

**Review by jinevening — APPROVED** [review]
LGTM

**Pull request merged**